crossplane/ext/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Raymond Lau"
+__copyright__ = "Copyright (C) Nginx, Inc. All rights reserved."
+__license__ = "Apache 2.0"
+__maintainer__ = "Raymond Lau"
+__email__ = "raymond.lau@nginx.com"
```

Comment:
We don't set these variables manually in the `__init__.py` files of crossplane since all that is handled in `setup.py` and the top-level files. In fact, most `__init__.py` files in this repo should be entirely empty (even without the coding things at the top). Anyway, to give yourself credit for your work you should add your name/email to the AUTHORS.rst file.

Reply:
ah gotcha, removing it